### DISCUSSION TITLE
ModuleNotFoundError: No module named 'atri_utils'

### DISCUSSION THREAD

**Discussion by Anurag-Sharma00** [Atri-Labs/atrilabs-engine#542] [Help: Installation & Start] [answered]
![Screenshot (4)](https://user-images.githubusercontent.com/105597363/201457962-ab8cbac5-b90b-4d93-8936-2eee234eb366.png)

**Comment by cruxcode** [comment, marked as answer]
It can be due to many reasons:

1. You might not be running inside the virtual environment you created for this project.
2. Pipenv failed to install `atri_utils` when you pressed `Build & Run`.

Solution:

Manually install the `atri_utils` package in your environment.
